src/hooks/use-deck-state.ts

```diff
@@ -37,60 +37,65 @@ type ReducerActions =
   | { type: 'COMMIT_TRANSITION'; payload?: DeckView }
   | { type: 'CANCEL_TRANSITION'; payload?: undefined };
 
+// If payload is null, we don't want to merge because it nulls out the whole
+// object.
+const mergeView = (view: DeckView, payload: Partial<DeckView>) =>
```

Comment:
@ryan-roemer  With the null-bug fixed, what do you think about keeping these `mergeView` changes? 
IMO, null checks are like novocain; it numbs things but you end up biting yourself.  I can revert them if you want.

Reply:
@scottrippey I'm down with whatever you feel is best -- I'm not that familiar with this code path / best practices and would love for you to just do what you think is best given the overall project and conventions..

Reply:
I rolled back this change, and verified that export mode is working correctly still 👍